README.md

```diff
@@ -2,43 +2,45 @@
 
 ---
 
-**Table of Contents** &nbsp;&nbsp; [Cockatrice](#cockatrice) | [Downloads](#downloads-) | [Get Involved](#get-involved-) | [Community](#community-resources) | [Translation](#translation-status-) | [Building](#building--) | [Running](#running) | [License](#license-)
+**Table of Contents** &nbsp;&nbsp; [Cockatrice](#cockatrice) | [Downloads](#downloads-) | [Get Involved](#get-involved-) | [Community](#community-resources) | [Translation](#translation-status-) | [Building](#building--) | [Running](#running) | [ReadMe Style Guide](#style-) | [License](#license-)
```

Comment:
The style anchor is broken - I think it needs to be `#readme-`

Reply:
Could the table of contents be split onto multiple lines? Right now it wraps at an odd place

Reply:
Good catch, thanks

Reply:
I've removed this entirely based on other suggestions